README.md

```diff
@@ -21,33 +60,47 @@ $ bower install gaia-components/gaia-button
 Add `disabled` attribute to disable the button.
 
 ```html
-<gaia-button disabled>button disabled</gaia-button>
+<gaia-button disabled><span data-l10n-id="disable">button disabled</span></gaia-button>
 ```
 
-### With icon
+### Circular
 
-Button Can show with icon (need import `gaia-icons`) at left
+Add `circular` attribute to turn button into circular form.
 
 ```html
-<gaia-button><i data-icon="back-light" aria-hidden="true"></i><span>Back</span></gaia-button>
+<gaia-button circular>
+  <span data-icon="camera" data-l10n-id="camera"/>
+</gaia-button>
 ```
 
-or right
 
-```html
-<gaia-button><span>Forward</span><i data-icon="forward-light" aria-hidden="true"></i></gaia-button>
-```
+## RTL
 
-Remember to add `aria-hidden="true"` when both icon and text existed for **Accessibility**. So the screen reader will read the button text and ignore the icon text.
+`gaia-button` is RTL ready. The icon and text position will be changed base on RTL settings.
 
-### Circular
 
-Add `circular` attribute to turn button into circular form.
+## Accessibility
+
+`aria-disabled="true"` is added automatically when button comes with `disabled` attribute. So the screen reader will read the button text and note it was disabled.
+
+* standalone icon butotn:
 
 ```html
-<gaia-button circular><i data-icon="camera"></i></gaia-button>
+<gaia-button>
+  <span data-l10n-id=""/>
```

Comment:
test with l10n.js 1.0x, the `data-icon` and the text defined in span will be both ignored when `data-l10n-id` is set in `gaia-button` tag-self.

Define `data-icon` and `data-l10n-id` in `gaia-button` tag self seems need rewrite current styles. I did not figure out how to do that thoroughly yet.